src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -87,25 +85,25 @@
             targetPlace = [...items].find((item, index) => index === draggedItemPosition);
         }
 
+        fieldNode.classList.add('ibexa-collapse--field-definition-highlight');
+        targetContainer.insertBefore(fieldNode, targetPlace);
+        removeDragPlaceholders();
```

Comment:
Removing drag placeholders should not be part of the logic related to creating field definition node.